.github/workflows/ci.yml

```diff
@@ -32,16 +31,59 @@ jobs:
           ~/.m2/repository
         key: maven-repo
 
-    - name: Initialize CodeQL
```

Comment:
Did you remove the CodeCQL Action?

Reply:
Yes, since we were only using an initializing step. The actual analysis step has never been there.
However, we can get this back in, this time with a separate analysis step.

Reply:
Has been added back again. This time with a separate analysis step.

Reply:
The analysis step was there.

Reply:
Sry, was comparing to an intermediate state. It's been there. So yes, all above is irrelevant.